Enhance Analytics Plugin

yzainee opened the pull request:
Changes added to support the new osio pipeline where the analytics flow
should not fail if the manifest files are not found.

The .hpi file was built from https://github.com/fabric8-analytics/fabric8-analytics-jenkins-plugin/pull/21

Tag: https://github.com/fabric8-analytics/fabric8-analytics-jenkins-plugin/releases/tag/0.3.3
Resolves: https://openshift.io/openshiftio/Openshift_io/plan/detail/1216

fabric8cd commented:
Good news @yzainee snapshot Jenkins image is available. `docker pull fabric8/jenkins-openshift:SNAPSHOT-PR-230-122`

fabric8cd commented:
Good news @yzainee snapshot Jenkins image is available. `docker pull fabric8/jenkins-openshift:SNAPSHOT-PR-230-126`

yzainee commented:
[test]

yzainee commented:
[test]

yzainee commented:
[retest]

hrishin commented:
[test]

fabric8cd commented:
Good news @yzainee snapshot Jenkins image is available. `docker pull fabric8/jenkins-openshift:SNAPSHOT-PR-230-127`

piyush-garg commented:
[test]

fabric8cd commented:
Good news @yzainee snapshot Jenkins image is available. `docker pull fabric8/jenkins-openshift:SNAPSHOT-PR-230-128`

piyush-garg commented:
@yzainee If your work is in progress, please add the DNM label

yzainee commented:
@piyush-garg No there is no more changes in the plugin. Was just adding some logs etc for more clarification as to what is sent to our plugin. The PR is good to be merged.

yzainee commented:
@piyush-garg @hrishin https://paste.centos.org/view/3435a166 Build Logs

piyush-garg commented:
Thanks, @yzainee I will also give it a try today and then merge.

piyush-garg commented:
@yzainee Can you please change commit message.

yzainee commented:
@piyush-garg done.

piyush-garg commented:
@yzainee In the other commit message, you have mentioned - removed version file. That is not part of this patch, Can you please check once again.

piyush-garg commented:
It will be better if you squash both the commits and provide a the commit message like the second commit. Thanks

yzainee commented:
@piyush-garg we used to update the version file as well earlier. After  discussion with @hrishin , I removed the version file which i had updated in the 1st commit as its no longer required.

piyush-garg commented:
[test]

fabric8cd commented:
Good news @yzainee snapshot Jenkins image is available. `docker pull fabric8/jenkins-openshift:SNAPSHOT-PR-230-129`
